src/app/loadout-drawer/LoadoutPopup.tsx

```diff
@@ -283,7 +300,22 @@ function LoadoutPopup({
           </>
         )}
 
-        {loadouts.map((loadout) => (
+        {totalLoadouts >= 10 && (
+          <li className={styles.menuItem}>
+            <form>
+              <AppIcon icon={searchIcon} />
+              <input
+                type="text"
```

Comment:
```suggestion
                type="text"
                autoFocus
```

Reply:
I'd rather not autofocus, partly because you don't always want the keyboard up and partly because iOS Safari does crazy things with layout on autofocus.

Reply:
I'll try it though, I guess I can always remove it afterwards.

Reply:
desktop-only would be fine. just feels like the speed offered by search is slowed by the need to aim for the text input among other buttons that wold be annoying to miss and click on. all other text fields in DIM have some lovely safe dead space around them. there's no trade off on desktop to add auto